README.rst

```diff
@@ -83,7 +83,7 @@ Import the pyunits library:
 
 Quantities can be instantiated with 1 of 3 methods:
 
-.. code:: python
+.. code:: pycon
```

Comment:
What is this?  It looks fine to me before.

Reply:
i added a code formatting using a tool called blacken, in that we would typically run that tool separately on your Python code files. So python code block was raising error, may be because of >>> symbol. May be some reformating of the rst files will remove this, i will try to revert, after the reformat

Reply:
Yes, there were syntax errors in the code blocks that I've now fixed up in the other PR.  You should not need this change here.